gulpfile.js

```diff
@@ -244,6 +244,8 @@ function dist() {
         './js/libraries/jquery-ui-1.11.4.min.js',
         './js/libraries/d3.min.js',
         './js/libraries/jquery.nouislider.all.min.js',
+        './js/libraries/i18n/i18next.js',
```

Comment:
Do we have to add copies of these libraries to our repository? I think it would be much preferrable if we leveraged the fact that we are already using `npm` and added them from https://www.npmjs.com/package/i18next .

Reply:
I think that the problem is that we are not packaging the node_modules, it is used only to generate the installer, not to execute, and if we change it chrome app does not support the majority of libraries.

Reply:
Yes we are packaging all node modules that are installed as `dependencies`. ;-) There just aren't any configured yet. What is the difference between the i18next library provided by npm and the static file that is contained in this pull request?